rpcs3/rpcs3qt/progress_dialog.cpp

```diff
@@ -42,6 +42,14 @@ void progress_dialog::SetValue(int progress)
 	QProgressDialog::setValue(value);
 }
 
+void progress_dialog::SignalFailure()
+{
+#ifdef _WIN32
+	m_tb_progress->stop();
+#endif
+	// TODO: Implement an equivalent for Linux, if possible
```

Comment:
Did you try with alert or what it was?

Reply:
I did not, I honestly never tested rpcs3 on Linux so I don't know what outcome to expect here.